src/calculateRentalCost.js

```diff
@@ -4,7 +4,18 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const perDay = 40;
+  let total = days * perDay;
+
+  if (days >= 7) {
+    total = total - 50;
```

Comment:
Replace the hard-coded value '7' with a named constant to clearly define what this magic number represents. Also, replace '50' with a named constant to indicate the discount amount for clarity and maintainability.